README.md

```diff
@@ -1,30 +1,48 @@
-# jiant v2 (Alpha)
-[![Generic badge](https://img.shields.io/badge/version-2.0.dev0-blue.svg)](https://shields.io/)
-[![codecov](https://codecov.io/gh/jiant-dev/jiant/branch/master/graph/badge.svg)](https://codecov.io/gh/jiant-dev/jiant)
+<div align="center">
+
+# `jiant`
+**The multitask and transfer learning toolkit for natural language processing research**
+
+[![Generic badge](https://img.shields.io/github/v/release/nyu-mll/jiant)](https://shields.io/)
+[![codecov](https://codecov.io/gh/nyu-mll/jiant/branch/master/graph/badge.svg)](https://codecov.io/gh/nyu-mll/jiant)
+[![CircleCI](https://circleci.com/gh/nyu-mll/jiant/tree/master.svg?style=shield)](https://circleci.com/gh/nyu-mll/jiant/tree/master)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
-`jiant` is a software toolkit for natural language processing research, designed to facilitate work on multitask learning and transfer learning for sentence understanding tasks. 
+</div>
+
+**Why should I use `jiant`?**
+- `jiant` supports [multitask learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_Multi_Task_Example.ipynb)
+- `jiant` supports [transfer learning](https://colab.research.google.com/github/nyu-mll/jiant/examples/notebooks/jiant_STILTs_Example.ipynb)
+- `jiant` supports these natural language understanding [tasks](./md_docs/tasks.md)
+- `jiant` supports these benchmarks:
+    - [GLUE](https://gluebenchmark.com) (TODO)
```

Comment:
If these TODOs will be there for the release, explain that it's only the tutorial, not the implementation, that's pending.

Reply:
These TODOs will be removed before opening a PR and updated to reflect the benchmark documentation location